skrifa/src/outline/cff/mod.rs

```diff
@@ -707,6 +709,36 @@ mod tests {
         assert!(svg.to_string().ends_with('Z'));
     }
 
+    /// Ensure we don't reject an empty Private DICT
+    #[test]
+    fn empty_private_dict() {
+        let font = FontRef::new(font_test_data::MATERIAL_ICONS_SUBSET).unwrap();
+        let common = OutlinesCommon::new(&font).unwrap();
+        let outlines = super::Outlines::new(&common).unwrap();
```

Comment:
So before, outlines came back as Err()? Do we want to access any contours here as part of the test?

Reply:
Constructing Outlines was fine, but making a Subfont would return an error so we couldn't generate an outline. This is checked, although indirectly, in the following test.

Reply:
Although the call to `Outlines::private_dict_range()` would fail in this case, which is checked here.